README.md

```diff
@@ -138,6 +139,21 @@ For information about maintenance and support for SDK major versions and their u
 * [AWS SDKs and Tools Maintenance Policy][maintenance-policy]
 * [AWS SDKs and Tools Version Support Matrix][version-matrix]
 
+## Getting Help
+
+The best way to interact with our team is through GitHub. You can
+[open an issue](https://github.com/aws/aws-sdk-java-v2/issues/new/choose) and choose
+from one of our templates for guidance, bug reports, or feature requests.
+
+You may find help from the community on [Stack Overflow](stack-overflow) with the
+tag [#aws-java-sdk-2.x](stack-overflow-tag) or on the
```

Comment:
The two StackOverflow links are not working, there's a typo in the reference link, it should be `[ ]` instead of `( )`

Reply:
Also, there's another SO tag: [[aws-sdk-java-2.0]](https://stackoverflow.com/questions/tagged/aws-sdk-java-2.0)